### PR TITLE
Discord Rich Presence show username toggle to settings

### PR DESCRIPTION
**Pull request by aapelix**
Added a setting to change if you want to show your Minecraft username in Discord rpc when playing

### CODE REVIEW

**Review by Goldsucher07 — COMMENTED** [review]
Code appears good

**Review by Goldsucher07 — APPROVED** [review]
Code appears good

File: .gitignore
Comment (chaush-server) [inline .gitignore:1]: ?

Comment (blryface) [reply .gitignore:1]: That's just how Git works?? What are you on about

Comment (Jamalam360) [reply .gitignore:1]: Package lock shouldn't be ignored generally

**Pull request closed without merging**